app/api-fetch/all-albums.ts

```
@@ -8,58 +9,94 @@
+        console.warn('API URL not set, using fallback URL');
     }
     try {
-        const storedAlbums = localStorage.getItem("albums");
-        const storedTime = localStorage.getItem("albumsTime");
-        
-        if (storedAlbums && storedTime && Date.now() - parseInt(storedTime) < 3600000) {
+        if (typeof window !== 'undefined') {
+            const storedAlbums = localStorage.getItem("albums");
+            const storedTime = localStorage.getItem("albumsTime");
+            
+            if (storedAlbums && storedTime && Date.now() - parseInt(storedTime) < 3600000) {
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)